pages/wi/partner-detail.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+
+import HeroSection from '../../components/PartnerDetail/HeroSection';
+import MainLogoSection from '../../components/PartnerDetail/MainLogoSection';
+import AboutGoalsSection from '../../components/PartnerDetail/AboutGoalsSection';
+import PresentationsJobsSection from '../../components/PartnerDetail/PresentationsJobsSection';
+
+const GET_PARTNER = gql`
+  query getPartner($partnerId: ID!) {
+    partner(id: $partnerId) {
+      id
+      year
+      companyName
+      companyLogo
+      heroImage
+      website
+      goals
+      aboutUs
+      contactNumber
+      linkedIn
+      github
+      youtube
+      instagram
+      twitter
+      facebook
+      twitch
+      chat
+      blog
+      vlog
+      jobListings {
+        id
+        title
+        description
+      }
+    }
+  }
+`;
+
+const MainDiv = styled.div`
+  padding-bottom: 4rem;
+`;
+
+const partnerDetail = ({ query }) => {
+  let partner = null;
+  const { loading, error, data } = useQuery(GET_PARTNER, {
+    variables: { partnerId: query.id },
+    onCompleted(d) {
+      partner = d.partner;
+      let hostName = new URL(partner.website).hostname;
+      if (hostName.toLowerCase().startsWith('www.')) {
+        hostName = hostName.replace('www.', '');
+      }
+      partner.hostName = hostName;
+      partner.whoToSayHiTo = [
```

Comment:
Is this the test data you are baking in cause we don't have the API wired up to feed it to you yet?

Reply:
correct, but for some reason it doesn't show up when deployed 🤷‍♂